core/src/com/cloud/agent/api/storage/PrimaryStorageDownloadCommand.java

```diff
@@ -87,4 +88,12 @@ public String getPrimaryStorageUrl() {
     public boolean executeInSequence() {
         return true;
     }
+
+    public String getNfsVersion() {
+        return nfsVersion;
+    }
+
+    public void setNfsVersion(String nfsVersion) {
+        this.nfsVersion = nfsVersion;
+    }
```

Comment:
This looks like a useful change to the storage API, but why is it necessary to implement OSPF?  Seems like it should be in a separate PR.